tools/Customization/DevHome.FileExplorerSourceControlIntegration/Program.cs

```diff
@@ -57,7 +57,7 @@ private static void HandleCOMServerActivation()
         Log.Information($"Activating COM Server");
         using var sourceControlProviderServer = new SourceControlProviderServer();
         var sourceControlProviderInstance = new SourceControlProvider();
-        var wrapper = new Microsoft.Internal.Windows.DevHome.Helpers.FileExplorer.PerFolderRootSelectorWrapper(sourceControlProviderInstance);
+        var wrapper = new Microsoft.Internal.Windows.DevHome.Helpers.FileExplorer.ExtraFolderPropertiesWrapper(sourceControlProviderInstance, sourceControlProviderInstance);

```

Comment:
Quick Question: Why is the same object instance sent twice as function argument? Is this related to the update in interface where one interface provides property information on folder and the other provides property information on files stored under the folder? Thanks!

Reply:
Yes, this is related to the interface update. One is for the legacy testing interface, and the other is for the new interface we'll use going forward. Rather than query at runtime for the existence of one of the interfaces (which could fail), I simply decided to provide two params to force the compiler to check for us. Once the helpers package can wean itself off the legacy testing interface, I'll delete the second param from that constructor and make the corresponding update here.